TESTING.md

```diff
@@ -0,0 +1,104 @@
+# Testing documentation
+
+To start the automatic test, you can use the following command:
+
+```bash
+python -m unittest discover -s tests
+```
+
+You also have the option to run automatic test in run_test.py or
+manually test the functionality with the frontend. [See frontend testing
+documentation](https://github.com/ai-cfia/nachet-frontend/blob/main/TESTING.md)
+
+To perform the following test, you will need the frontend repository for the
+Nachet Interactive's application. The frontend can be found at: [Nachet Frontend
+GitHub Repository](https://github.com/ai-cfia/nachet-frontend).
+
+You will also need the list of the available pipelines. The list can be found
+[here](https://github.com/ai-cfia/nachet-backend/blob/51-implementing-2-models/docs/nachet-inference-documentation.md#available-version-of-the-json-file).
+
+---
+
+## Test Case: Populate model selection with pipelines information
+
+**Objective**: Verify that the model selection component gets populated with the
+pipeline information.
+
+**Preconditions:**
+
+- [ ] Nachet backend is set up and running. Use the command `hypercorn -b :8080
+  app:app` to start the quartz server.
+- [ ] The environment variables are all set.
+- [ ] :exclamation: The frontend is not running yet
+
+**Test Steps:**
+
+1. Start the frontend application
+1. Click on the model selection button
+1. Validate that the current pipeline are selectable.
+
+**Expected Results:**
+
+- [ ] If a problem occur while retrieving the data, a error should have stop the
+  server to start.
+- [ ] If a problem occur while retrieving the data, but no error was raised, the
+  model selection component will be empty.
+- [ ] If everything went correctly while retrieving the data, the model
+  selection component will display the pipeline metadata.

```

Comment:
If everything went correctly while retrieving the data, the model selection component should display the pipeline metadata.